pkg/config_legacy/configenv/production.go

```diff
@@ -125,7 +125,6 @@ var ProductionComputeConfig = types.ComputeConfig{
 		InfoUpdateFrequency:     types.Duration(60 * time.Second),
 		ResourceUpdateFrequency: types.Duration(30 * time.Second),
 		HeartbeatFrequency:      types.Duration(15 * time.Second),
-		HeartbeatTopic:          "heartbeat",
```

Comment:
Let's revert all changes made to config_legacy package.

Reply:
Done